Stripe/STPPaymentMethodType.m

```diff
@@ -65,15 +65,15 @@ @interface STPPaymentMethodTypeSEPADebit : STPPaymentMethodType
 @implementation STPPaymentMethodTypeSEPADebit
 
 - (UIImage *)paymentMethodImage {
-    return [STPImageLibrary addIcon];
+    return [STPImageLibrary sepaIcon];
 }
 
 - (UIImage *)paymentMethodTemplateImage {
-    return [STPImageLibrary addIcon];
+    return [STPImageLibrary sepaIcon];
 }
 
 - (NSString *)paymentMethodLabel {
-    return STPLocalizedString(@"New SEPA Debit", @"Label for 'new sepa debit' payment method field") ;;
+    return STPLocalizedString(@"Direct Debit Account", @"Label for SEPA Direct Debit payment method field") ;;
```

Comment:
Do we need to alter the voiceover text for this to make it clear it is SEPA for blind users? (as it might not be clear if you can't see the associated icon)